plugins/modules/device_credential_workflow_manager.py

```diff
@@ -1192,416 +1180,432 @@ def get_snmpV3_params(self, snmpV3Details):
                         "authType": item.get("authType"),
                         "privacyType": item.get("privacyType")
                     })
-                snmpV3.append(value)
-        return snmpV3
+                snmp_v3.append(value)
+        return snmp_v3
 
-    def get_cli_credentials(self, CredentialDetails, global_credentials):
+    def get_cli_credentials(self, credential_details, global_credentials):
         """
         Get the current CLI Credential from
         Cisco Catalyst Center based on the provided playbook details.
         Check this API using the check_return_status.
 
         Parameters:
-            CredentialDetails (dict) - Playbook details containing Global Device Credentials.
+            credential_details (dict) - Playbook details containing Global Device Credentials.
             global_credentials (dict) - All global device credentials details.
 
         Returns:
-            cliDetails (List) - The current CLI credentials.
+            cli__details (List) - The current CLI credentials.
         """
 
         # playbook CLI Credential details
-        all_CLI = CredentialDetails.get("cli_credential")
+        all_cli = credential_details.get("cli_credential")
         # All CLI details from Cisco Catalyst Center
         cli_details = global_credentials.get("cliCredential")
+
         # Cisco Catalyst Center details for the CLI Credential given in the playbook
-        cliDetails = []
-        if all_CLI and cli_details:
-            for cliCredential in all_CLI:
-                cliDetail = None
-                cliId = cliCredential.get("id")
-                if cliId:
-                    cliDetail = get_dict_result(cli_details, "id", cliId)
-                    if not cliDetail:
+        cli__details = []
```

Comment:
Double underscores ??  Can you rename to a meaningful one?
How can one differentiate cli_details and cli__details?